editor/editor_help.cpp

```diff
@@ -2384,6 +2387,7 @@ static void _add_text_to_rt(const String &p_bbcode, RichTextLabel *p_rt, Control
 	List<String> tag_stack;
 	bool code_tag = false;
 	bool codeblock_tag = false;
+	StringBuilder codeblock_text; // More efficient to append with.
```

Comment:
I think the comment is not necessary, as StringBuilder's purpose is self-explanatory.

Reply:
Just like that gone in a flash.